### PR TITLE
jv-spring-rest-web-practice task completed

### PR DESCRIPTION
**Pull request by Gleb-Kreshchuk**

### CODE REVIEW

**Review by kvoytikh — APPROVED** [review]

File: src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package mate.academy.spring.model.dto.response;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class OrderResponseDto {
+    private Long id;
+    private List<Long> ticketIds;
+    private LocalDateTime orderDate;
```
Comment (Iryna-U) [inline src/main/java/mate/academy/spring/model/dto/response/OrderResponseDto.java:9]: let's return in response some simple data types, like primitives, wrappers and String. smth like LocalDateTime will be hard to parse

**Review by Iryna-U — CHANGES_REQUESTED** [review]

**Review by Iryna-U — APPROVED** [review]
